src/main/resources/assets/natura/lang/de_DE.lang

```diff
@@ -0,0 +1,582 @@
+##########################
+#    Overworld Blocks    #
+##########################
+
+tile.natura.clouds.white.name=Wolke
+tile.natura.clouds.white.tooltip=So leicht und flauschig!\nRettet Dich, wenn Du fällst
+tile.natura.clouds.dark.name=Dunkle Wolke
+tile.natura.clouds.dark.tooltip=So leicht und flauschig!\nVerursacht Blitzeinschläge
+tile.natura.clouds.ash.name=Aschewolke
+tile.natura.clouds.ash.tooltip=So leicht und flauschig!\n3x3 Verwandelt sich in Holzkohle
+tile.natura.clouds.sulfur.name=Schwefelwolke
+tile.natura.clouds.sulfur.tooltip=So leicht und flauschig!\n2x2 Verwandelt sich in Schwefel
+
+tile.natura.overworld_logs.maple.name=Ahornholz
+tile.natura.overworld_logs.maple.tooltip=Etwas süßlich
+tile.natura.overworld_logs.silverbell.name=Silberglöckchenholz
+tile.natura.overworld_logs.silverbell.tooltip=Silberne Glocken
+tile.natura.overworld_logs.amaranth.name=Amaranthholz
+tile.natura.overworld_logs.amaranth.tooltip=Herz aus Holz
+tile.natura.overworld_logs.tiger.name=Tigerholz
+tile.natura.overworld_logs.tiger.tooltip=Wildgetreide
+
+tile.natura.overworld_logs2.willow.name=Weidenholz
+tile.natura.overworld_logs2.willow.tooltip=Der Weinende
+tile.natura.overworld_logs2.eucalyptus.name=Eukalyptusholz
+tile.natura.overworld_logs2.eucalyptus.tooltip=Das rosa Holz
+tile.natura.overworld_logs2.sakura.name=Sakuraholz
+tile.natura.overworld_logs2.sakura.tooltip=Blühende Kirsche
+tile.natura.overworld_logs2.hopseed.name=Hopfensaatholz
```

Comment:
I think those might be called "Hop-Busch" in German, but I am not sure.

Reply:
I had a look at Wikipedia, it's called "Hopfenbusch" in German, so you were very close. ;)
https://de.wikipedia.org/wiki/Dodonaea_viscosa#Namensgebung

I'll correct it.

Reply:
That just says it's the literal translation, not what is actually used. I've seen "Hop-Busch" on the internet before, but I may not be right.

Reply:
Yeah, but I have literally no clue what a trivial German name for it may be.
If you wish, I'll name it "Hop-Busch", probably just a few people would notice a problem anyway.

But I'll do as you wish. :D